tests/component/Map.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from "@testing-library/react"
+import Map from "../../components/Map"
+
+describe("Map", () => {
+  it("renders the Map component with the required props", () => {
+    const mapProps = {
+      center: { lat: 47.0254538, lng: -91.6761874 },
+      zoom: 14,
+      id: "mapView"
+    }
+
+    render(<Map {...mapProps} />)
+
+    expect(screen.getByTestId("mapView")).toBeInTheDocument()
```

Comment:
You should try to use `role` or a label here. `getByTestId` doesn't offer any useful context to the user whereas the first two would help your app be more accessible.